.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+---
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 0 * * 1,2,3,4,5'
+  workflow_dispatch:
+
+name: docker
+
+jobs:
+  docker:
+    name: Docker actions
+    uses: RMI-PACTA/actions/.github/workflows/docker.yml@main
+    with:
+      do-check-r-sysdeps: false
+
+  gh-pages:
+    name: gh-pages
+    uses: ./.github/workflows/gh-pages.yml
+    needs: [docker]
+    secrets: inherit
+    # strategy:
+    #   fail-fast: false
+    #   matrix:
+    #     config-name:
+    #       - default_2022Q4
+    #       - default_2023Q4
+    #       - full_params_2022Q4
+    #       - full_params_2023Q4
```

Comment:
NIT: Consider removing commented out/ dead code before merging to `main`

Reply:
Hmm. most of this is going to come right back / be uncommented when I start work on #34, which is why I'm not too worried about it right now. But yeah, I can pull these out.